modelling/src/main/java/org/axonframework/modelling/saga/metamodel/SagaModel.java

```diff
@@ -58,6 +61,29 @@ default boolean hasHandlerMethod(EventMessage<?> eventMessage) {
         return !findHandlerMethods(eventMessage).isEmpty();
     }
 
+    /**
+     * Returns an Interceptor Chain of annotated interceptor methods. The given chain will invoke all relevant
+     * interceptors in an order defined by the handler definition.
+     *
+     * @return an interceptor chain that invokes the interceptor handlers
+     */
+    default MessageHandlerInterceptorMemberChain<T> chainedInterceptor() {
```

Comment:
I get why you've added the `MessageHandlerInterceptorMemberChain` to this interface after a little playing around, but I don't think it necessarily belongs here.

Mainly because I am not overly convinced it's a task the model instance should have.

Could we perhaps instead add this method to the `SagaMetaModelFactory`?
The factory's exposed on the API of the `SagaModel`, so we can retrieve the chain from there.

Working a little on gut feeling here, though. So please, do give your opinion on the matter too, @gklijs.

Reply:
I don't have a strong take, but it does seem nicer to keep the `SagaModel` clean. Going to change the PR.